### DISCUSSION TITLE
add cli to module

### DISCUSSION THREAD

**Discussion by l1t1** [chdb-io/chdb#109] [Ideas]
now we must download and set the file `chdb-cli.py` manually.
it will be easy to if it could run as `python -m chdb cli` after `pip install chdb`.

**Comment by auxten** [comment]
I think it's a good idea. But this will introduce chdb more requirements. Maybe, putting chdb-cli into pypi. `pip install chdb-cli` is a better idea?

**Reply by l1t1** [reply]
good
